sources/core/Stride.Core.Serialization/Serialization/Contents/ContentManager.cs

```diff
@@ -402,6 +402,19 @@ private object DeserializeObject(string initialUrl, string newUrl, Type type, ob
 
             return result;
         }
+        
+        public Type GetType(string url)
+        {
+            if(LoadedAssetUrls.TryGetValue(url, out var reference))
```

Comment:
Cool, but prefer null-conditional operators to prevent nested loops 👀. Here is my suggestion
```suggestion
            public Type GetType(string url)
        {
            _ = LoadedAssetUrls.TryGetValue(url, out var reference); 
            return reference.Object?.GetType();
        }
```

Reply:
Updated!